base/linalg/triangular.jl

```diff
@@ -319,7 +343,7 @@ end
 function A_mul_B!(A::UpperTriangular, B::StridedVecOrMat)
     m, n = size(B, 1), size(B, 2)
     if m != size(A, 1)
-        throw(DimensionMismatch("left and right hand side does not fit"))
+        throw(DimensionMismatch("B needs first dimension of size $(size(A,1)), has size $m"))
```

Comment:
Usually, I try to use a more descriptive name (like left and right hand side) instead of the name used inside the function. These names are not always that easy to understand for users even though things might have improved with the new documentation system.